lib/src/cloud_firestore_mocks_base.dart

```diff
@@ -106,8 +108,18 @@ class MockFirestoreInstance extends Mock implements Firestore {
   }
 
   String dump() {
+    final copy = deepCopy(_root);
```

Comment:
Can you add a comment to this loop? Something like:

```
// Place each document at the right place in the collection hierarchy.
```

Reply:
Done!